package.nls.json

```diff
@@ -70,8 +71,14 @@
     "AWS.samcli.detect.settings.not.updated": "No settings changes necessary.",
     "AWS.samcli.configured.location": "Configured SAM CLI Location: {0}",
     "AWS.samcli.error.notFound": "Unable to find the SAM CLI, which is required to create new Serverless Applications and debug them locally. If you have already installed the SAM CLI, update your User Settings by locating it.",
+    "AWS.samcli.error.notFound.brief": "Could not get SAM CLI location",
+    "AWS.samcli.notification.version.invalid": "Your SAM CLI version {0} does not meet requirements ({1} - {2}). {3}",
```

Comment:
In JetBrains we do "min ≤ ver < max". Can we do the same or otherwise communicate that it's [min, max)?